pkg/controller/flinkapplication/flink_state_machine.go

```diff
@@ -28,9 +28,10 @@ import (
 )
 
 const (
-	jobFinalizer    = "job.finalizers.flink.k8s.io"
-	statusChanged   = true
-	statusUnchanged = false
+	jobFinalizer                  = "job.finalizers.flink.k8s.io"
+	statusChanged                 = true
+	statusUnchanged               = false
+	jobVertexStateTimeoutInMinute = 3
```

Comment:
Can this be part of the config so that it can be set through configmap?

Reply:
Yes, it could be. But I think even we set it through configmap, we still need a default value in code in case that this key is not set in configMap. If we have the use case that flinkk8soperator user want to override the default value later, we can consider to add it then.

Reply:
Correct. You can add the default value as well. 

https://github.com/lyft/flinkk8soperator/blob/12959ee99d4fd7edc5f801e732a8533a2bd82d53/pkg/controller/config/config.go#L1